README.rst

```diff
@@ -46,7 +46,13 @@ To build the PDF form of the documentation simply execute the following command:
 
   make latexpdf
 
-Note: Creating the PDF form of the documentation requires that latexmk is available, information on latexmk can be found at https://ctan.org/pkg/latexmk/.
+Note: Creating the PDF form of the documentation requires that some or all of a tool like Windows/MikTeX, Windows/TeXlive, Linux/TeXlive, and macOS/MacTeX is available to the Sphinx build tool.
```

Comment:
_and_ `latexmk`. It's a perl script that calls some underlying tex tool, but you need to install it separately. I just had to, anyway, despite having texlive and sphinx etc. installed

Reply:
https://github.com/sphinx-doc/sphinx/issues/3543
https://packages.ubuntu.com/xenial/latexmk
https://fedora.pkgs.org/31/fedora-x86_64/latexmk-4.65-2.fc31.noarch.rpm.html

Reply:
Did you install TeXlive as a complete package or did you pull it in in parts?  The macOS MacTeX has it bundled in. Will add it.

Reply:
Fedora or maybe texlive themselves decided a few years back that the pre-installed packages should be quite a minimal set. Thinking about it a bit more, maybe this is part of https://github.com/cellml/cellml-specification/issues/154 and we should look for some way to find out which latex packages people need?
(_or_ assume that people who want to compile latex have done so before)

Reply:
Shouldn't block this PR anyway!